configure.ac

```diff
@@ -289,6 +289,9 @@ AS_IF([test "x$enable_asm" != xno], [
             ])
             rm conftest.asm conftest.o > /dev/null 2>&1
         ])
+    ], [
+        AC_MSG_NOTICE(Assembly optimizations are not yet supported on this architecture; disabling.)
+        enable_asm=no
```

Comment:
This was effectively already done in f975de5d96aafd804c66ea79f0780ce2b30e1928 and can be dropped from this patch series; look ~50 lines up at the initial architecture check.

Reply:
Ah, I have a feeling that I've already seen that somewhere, but rebase went off without a hitch so I thought that it's from some unmerged PR.